### DISCUSSION TITLE
Would you be willing to document what `None` means?

### DISCUSSION THREAD

**Discussion by stdedos** [psf/pypistats.org#64] [Q&A] [answered]
It is my first time accessing this site. I'd be nice if there was something written about this 🙏

**Comment by hugovk** [comment, marked as answer]
From the original mailing list announcement:

> Oh, one additional tidbit of information- Most of the data is coming from parsing user agents, which means that the ability to get this information depends a lot on what is actually downloading the file. Generally most of the information is only available on pip (and newer pip versions added progressively more information). In cases where we didn’t know we have inserted (null) values (such as bandersnatch mirroring and not knowing what the “python version” is).
> 
> This means that you’ll often times see (null) values in fields, which just represents downloads using something where we couldn’t determine that information from.

https://mail.python.org/pipermail/distutils-sig/2016-May/028987.html
